src/export-members.ts

```diff
@@ -2,7 +2,17 @@ import snapshot from '@snapshot-labs/snapshot.js'
 import { Networks } from './entities/Networks'
 import { SnapshotSpace } from './interfaces/GovernanceProposal'
 import { DelegationInfo, MemberInfo, STRATEGIES, Vote } from './interfaces/Members'
-import { errorToRollbar, fetchDelegations, fetchGraphQLCondition, flattenArray, parseVP, saveToCSV, saveToJSON, snapshotUrl, splitArray } from './utils'
+import {
+  fetchDelegations,
+  fetchGraphQLCondition,
+  flattenArray,
+  reportToRollbarAndThrow,
+  saveToCSV,
+  saveToJSON,
+  snapshotUrl,
+  splitArray
+} from './utils'
+import { parseVP } from './vp-utils'
 
 const MAX_RETRIES = 10
 
```

Comment:
Once the new strategies are added, in `fetchSnapshotScores` the 'snapshotScores' initialization could be like this:

`STRATEGIES.map<Record<string, number>>((strategy) => ({}))`

instead of a fixed array